lib/colorado_lottery.rb

```diff
@@ -0,0 +1,60 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants, :winners, :current_contestants
+  def initialize
+    @registered_contestants = Hash.new { |hash, key| hash[key] = [] }
+    @winners = []
+    @current_contestants = Hash.new { |hash, key| hash[key] = [] }
+  end
+
+  def interested_and_18?(contestant, game)
+    contestant.game_interests.include?(game.name) && contestant.age >= 18
+  end
+
+  def can_register?(contestant, game)
+    interested_and_18?(contestant, game) &&
```

Comment:
Great job using the return value of interested_and_18? in conjunction with the new conditions.